<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="EPBYMINW0635" name="com.epam.tat.calctest.testng.TstSqrt" tests="7" failures="1" timestamp="01 Oct 2018 12:18:51 GMT" time="0.010" errors="0">
  <testcase name="sqrtTest" time="0.001" classname="com.epam.tat.calctest.testng.TstSqrt"/>
  <testcase name="sqrtTest" time="0.000" classname="com.epam.tat.calctest.testng.TstSqrt"/>
  <testcase name="sqrtTest" time="0.005" classname="com.epam.tat.calctest.testng.TstSqrt"/>
  <testcase name="sqrtTest" time="0.001" classname="com.epam.tat.calctest.testng.TstSqrt"/>
  <testcase name="sqrtTest" time="0.001" classname="com.epam.tat.calctest.testng.TstSqrt">
    <failure type="java.lang.AssertionError" message="Invalid result expected [NaN] but found [1.0]">
      <![CDATA[java.lang.AssertionError: Invalid result expected [NaN] but found [1.0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at com.epam.tat.calctest.testng.TstSqrt.sqrtTest(TstSqrt.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- sqrtTest -->
  <testcase name="sqrtTest" time="0.002" classname="com.epam.tat.calctest.testng.TstSqrt"/>
  <testcase name="negativeArgumentTest" classname="com.epam.tat.calctest.testng.TstSqrt">
    <skipped/>
  </testcase> <!-- negativeArgumentTest -->
</testsuite> <!-- com.epam.tat.calctest.testng.TstSqrt -->
